src/main/java/com/google/sps/servlets/SortedMarkersServlet.java

```diff
@@ -59,23 +72,28 @@ private String getParameter(HttpServletRequest request, String name, String defa
     try {
         ArrayList<EventMarker> mapEvents = new ArrayList<EventMarker>();
         EventsServlet eventsServlet = new EventsServlet();
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Entity group = datastore.get(KeyFactory.createKey(GROUP_KIND, String.valueOf(groupId)));
         
-        // TODO: Convert the hardcoded calendarID and groupName when getEventsList() param is changed to groupID
-        Events calEvents = eventsServlet.getEventsList("fk6u4m5isbl8i6cj1io1pkpli4@group.calendar.google.com");
+        // TODO: Relplace getEventsList() param to groupId once Dylan's PR is merged
```

Comment:
two small things:
(1) "Replace" is spelled wrong :) 
(2) Instead of calling it "Dylan's PR", it's better to describe what the PR is. E.g. you could put "Replace getEventsList() param once the Event Param code has been merged". 
As the reader, I don't know what Dylan's PR is doing. Also, at work, people could leave the team, at which point the ldap or name isn't as useful to us anymore.

Reply:
OK makes sense!